tests/api/test_libraries_rest.py

```diff
@@ -303,7 +303,7 @@ def test_library_secure_api_update(client, lib_fully,
         data=json.dumps(data),
         headers=json_header
     )
-    assert res.status_code == 200
```

Comment:
For each test can you put some comment why it is not possible? Can you also add a sucess full test: a system librarian modify a library. Can you do the same for the other tests of this commit?